pkg/query/paging/README.md

```diff
@@ -101,26 +102,38 @@ func ValidateAndApplyPagingRules(model PagingSettingsInterface, request *Request
 ValidateAndApplyPagingRules performs a validation of the original request and adds correct the correct values \(defaults\) if needed
 
 <a name="Response"></a>
-## type [Response](<https://github.com/greenbone/opensight-golang-libraries/blob/main/pkg/query/paging/response.go#L6-L10>)
+## type [Response](<https://github.com/greenbone/opensight-golang-libraries/blob/main/pkg/query/paging/response.go#L8-L13>)
 
 Response represents a response object containing information about pagination and total count of records.
 
-- PageIndex: The index of the page \(starting from 0\).
-- PageSize: The number of records per page.
+- PageIndex: The index of the page \(starting from 0\). This is required.
+- PageSize: The number of records per page. This is required.
+- TotalDisplayableResults: The total number of results that can be displayed. This is required.
+- TotalResults: The total number of results, including those that may not be displayed. This is optional and must not be set if the value does not differ from \`TotalDisplayableResults\`
 
 ```go
 type Response struct {
-    PageIndex     int    `json:"index"`
-    PageSize      int    `json:"size"`
-    TotalRowCount uint64 `json:"total"`
+    PageIndex               int    `json:"index" binding:"required"`
+    PageSize                int    `json:"size" binding:"required"`
+    TotalDisplayableResults uint64 `json:"totalDisplayableResults" binding:"required"`
```

Comment:
Not needed as this is the length of the result returned by the query and is not related to paging

Reply:
See comment on the need for `totalDisplayableResults` https://github.com/greenbone/opensight-golang-libraries/pull/94#discussion_r1687857980